guides/source/plugins.md

```diff
@@ -382,7 +391,13 @@ module Yaffle
   end
 end
 
-ActiveRecord::Base.include(Yaffle::ActsAsYaffle)
+# test/dummy/app/models/application_record.rb
+
+class ApplicationRecord < ActiveRecord::Base
```

Comment:
I tried to sell this, so I can hint the plugin folks. Tell me if you think it's an overkill.